mosu-app/src/entities/exam/ui/RegisteredExamInfoCard.tsx

```diff
@@ -26,6 +33,11 @@ export const RegisteredExamInfoCard = ({
                     "[&_p>span:nth-child(2)]:text-[#D9D9D9] [&_p>span:nth-child(2)]:mx-2",
                 )}
             >
+                <p>
+                    <span>수험번호</span>
+                    <span>|</span>
+                    <span>{examNumber}</span>
+                </p>
```

Comment:
```suggestion
                <p>
                    <span>{TagType}</span>
                    <span>|</span>
                    <span>{TagValue}</span>
                </p>
```
저 부분이 반복되는 것 같은데 공용으로 빼는건 어떻게 생각하세요?
몇 개 없어서 저렇게 쓰는거도 상관없어 보이는데

Reply:
아직은 말씀해주신대로 해당 부분을 재사용하는 코드가 몇 개 없어서 공용으로 빼지 않았는데 추후 더 재사용 가능성이 있으면 해당 피드백 고려하겠습니다. 피드백 감사합니다!